lib/rubocop/cop/mixin/autocorrector.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # extend this module to signal autocorrection support
+    module Autocorrector
```

Comment:
Btw, let's name this `AutoCorrector`, as it looks weird to me without the CamelCase. :-)

Reply:
Sure, I'm fine with it either way. Note that `auto-correct` is used for CLI options and rake task, while most uses in the sources use `autocorrect`.